docs/config.md

```diff
@@ -395,10 +419,21 @@ FSI
 
 Use the option `namePointDisplacement unused;` for solvers that do not create a pointDisplacement field, such as the RBFMeshMotionSolver.
 
-#### Debugging
+For FF simulations:
+
+```c++
+FF
+{
+  // Velocity
+  nameU U;
+  // Pressure
+  nameP p;
+}
+```
 
-The adapter also recognizes a few more parameters, which are mainly used in debugging or development.
-These are optional and expect a `true` or a `false` value. Some or all of these options may be removed in the future.
+Note that the adapter does not automatically adapt the pressure name for compressible solvers.
```

Comment:
This sentence is a duplicate. On purpose?

Reply:
Yes, because the one is in the FSI section, and the other in the FF section. But I will see how I can formulate this better.

Reply:
Clarified, linking to https://www.openfoam.com/documentation/guides/latest/doc/guide-applications-solvers-variable-transform-p-rgh.html